eclipse-scout-core/src/table/Table.ts

```diff
@@ -2289,13 +2300,6 @@ export class Table extends Widget implements TableModel, Filterable<TableRow> {
     });
   }
 
-  protected _triggerRowAction(row: TableRow, column: Column<any>) {
```

Comment:
Removing this and adding the new property needs documentation in the release notes.